src/components/index-page/IndexBuiltForYou.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import cornerOrnament from "../../assets/images/corner_ornament.png";
+import "./css/index-built-for-you.css";
+
+export const IndexBuiltForYou = () => (
+  <section className="index-built-for-you">
```

Comment:
let's not assume that this content will always appear on the Index - I recommend we drop the prefix in the css